include/vcpkg/install.h

```diff
@@ -25,15 +27,19 @@ namespace vcpkg::Install
 
     struct SpecSummary
```

Comment:
Long term, I think SpecSummary should be "freestanding" and not require the action plan that produced it to be held "alive" in order to be meaningful but that turned out to be more effort than I was willing to spend here. This at least goes somewhat in that direction.